packages/xml-lsp-server/test/server.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from "chai";
+
+import { validateDocument, SYNTAX_ERROR_MSG } from "../src/serverHelper";
+
+import {
+  TextDocument,
+  Diagnostic,
+  DiagnosticSeverity,
+  Position
+} from "vscode-languageserver";
+
+describe("Test XML Language Server", () => {
```

Comment:
All the `describe` and `it` need text in BDD style.
- https://mochajs.org/#bdd
- [Background Info](https://en.wikipedia.org/wiki/Behavior-driven_development#Story_versus_specification)

```javascript
describe("the XML Language Server")
  it("will detect a parsing error in an xml document")
  it("will detect a lexing error in an xml document")
  it("will not detect any error in a none XML document")
```